testing/python_lib/build_config.py

```diff
@@ -201,6 +202,24 @@ def create_corp_faucet_config(self):
         return FaucetConfig(dps=dps, version=2)
 
 
+def cleanup_keys(map_in):
+    """proto_dict converstaion converts int keys to strings, which causes problems with faucet."""
+    map_out = {}
+    for key in map_in:
+        map_out[int(key)] = map_in[key]
+    return map_out
+
+
+def cleanup_config(config_map):
+    """cleanup the necessary dict entries"""
+    for dp_id in config_map['dps']:
+        bad_map = config_map['dps'][dp_id]['interfaces']
+        config_map['dps'][dp_id]['interfaces'] = cleanup_keys(bad_map)
+    if 'vlans' in config_map:
+        config_map['vlans'] = cleanup_keys(config_map['vlans'])
```

Comment:
It's really not necessary to have another function for this, you can replace cleanup_keys with this line `dict(iter((int(k), v) for k, v in config_map['vlans'].items()))`